packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -367,6 +380,10 @@ describeFullCompat("Flushing ops", (getTestObjectProvider) => {
         }
 
         describe("Automatic flushing of batches via orderSequentially", () => {
```

Comment:
Similar to my comment above, these tests need to work for Immediate FlushMode as well.